src/Encoding/FilteredStream.php

```diff
@@ -24,16 +24,22 @@
 
     /**
      * @var resource
+     *
+     * @deprecated since version 1.5, will be removed in 2.0
      */
     protected $readFilter;
 
     /**
      * @var callable
+     *
+     * @deprecated since version 1.5, will be removed in 2.0
      */
     protected $writeFilterCallback;
 
     /**
      * @var resource
+     *
+     * @deprecated since version 1.5, will be removed in 2.0
      */
     protected $writeFilter;
```

Comment:
i tend to agree, but lets maybe leave as is, there is stuff to remove when we go 2.0 anyways so we can play it safe. does not add cost to leave those in